pages/Copilot/EKS/copilot-eks-clusters-dashboard.md

```diff
@@ -0,0 +1,149 @@
+
+As shown in the screen recording above, the EKS page features various cards and charts displaying key information about your EKS clusters. This page aggregates metrics across all your clusters that are correctly configured.
+
+{%include important.html content="Cost Breakdown, Cost Over Time, and Container Utilization metrics require Business Contexts+ (BC+). BC+ helps you understand and allocate 100% of your AWS costs, from your largest resources down to individual container costs. Click [here](https://www.nops.io/blog/introducing-business-contexts/) to learn more about BC+."%}
+
+Here's an explanation of each card:
+
+1. **Cost Breakdown**
+
+![copilot-eks-cost-breakdown.png](/tmpimg/copilot-eks-cost-breakdown.png)
+
+The Cost Breakdown chart on the EKS page provides a clear view of how costs are distributed across different components of your EKS clusters. This pie chart categorizes costs into several key areas, helping you understand where your budget is allocated. Here’s what each category represents:
+
+* **CPU**: This section shows the costs associated with the compute instances used by your clusters. It includes expenses related to the number of virtual CPUs and the types of instances in use.
+
+* **GPU**: This part highlights the costs for GPU resources utilized by your clusters. If your workloads use GPU instances, their associated costs are reflected here.
+
+* **Memory**: This category covers the expenses related to the amount of RAM allocated to your clusters. It includes costs based on memory usage.
+
+* **Storage**: This section represents the costs for various storage services, such as volumes and snapshots. It includes charges for storing and managing data within your clusters.
+
+* **Network**: This part accounts for costs associated with data transfer, NAT gateways, and load balancers. It includes charges for network-related services and data movement.
+
+* **Extended Support**: This category reflects additional charges incurred when your EKS clusters are not updated to the latest version. AWS applies these charges as a penalty for running outdated versions, which can include costs for extended support services.
+
+* **Control Plane**: This section includes costs not classified under the above categories. It covers miscellaneous expenses related to the management and operation of your clusters.
+
+2. **Number of Clusters**
+
+![copilot-eks-number-of-clusters.png](/tmpimg/copilot-eks-number-of-clusters.png)
+
+The Number of Clusters card on the EKS page provides an overview of your clusters and their configuration status. This card helps you quickly assess how many clusters you have and their current setup. Here’s a breakdown of what each status means:
+* **Configured**:
+    - **Fully**: Displays the number of clusters where all necessary agents and configurations are installed. For example, "Fully 5" indicates that 5 clusters have complete setup with all required agents and configurations.
```

Comment:
same thing here @jp-tavarez - Displays the number of clusters where all necessary nOps agents and configurations are installed on the cluster. For example, "Fully 5" indicates that 5 clusters have complete setup with all required agents and configurations on nOps Compute Copilot.